aptos-move/framework/src/extended_checks.rs

```diff
@@ -38,6 +39,7 @@ const LEGACY_ENTRY_FUN_ATTRIBUTE: &str = "legacy_entry_fun";
 const ERROR_PREFIX: &str = "E";
 const EVENT_STRUCT_ATTRIBUTE: &str = "event";
 const RANDOMNESS_ATTRIBUTE: &str = "randomness";
+const RANDOMNESS_MAX_GAS_CLAIM: &str = "max_gas";
```

Comment:
this may not be very clear to user, how about "gas_deposit"?

Reply:
no way the keyword itself will be self-explanatory. we should rely on the documentation?

Reply:
we should have both, but isn't "gas_deposit" better? maybe it's just me

Reply:
`gas_deposit` sounds more like implementation details of undergasing prevention, which dev doesn't need to know?

Reply:
this is the amount that user needs to deposit? but I don't feel very strongly

Reply:
yeah, let's just explain more in the doc.